src/test/resources/simple/not-allowed-url-bag/fetch.txt

```diff
@@ -0,0 +1 @@
+ftp://dans.knaw.nl/nl/over/organisatie-beleid/informatiemateriaal/Software_Sustainability_DANS_NLeSC_2016.pdf  555292  data/brochure/Software_Sustainability_DANS_NLeSC_2016.pdf
```

Comment:
That's what is called discipline! We ought to keep each other to the highest standards, especially regarding this kind of things. But to be honest with you, I have not the slightest clue what differences there are between the various bags in the various folders/sub-folders. Folder-names do not solve this issue completely. With some prior knowledge I understand that `invalid-url-bag` has something to do with `fetch.txt` (or maybe not?), but where can I find the difference between `example-bag` and `simple-bag` unless starting a new carrier as a detective? That's why we need this documentation on root level!